core/base/src/TString.cxx

```diff
@@ -254,12 +254,12 @@ char *TString::Init(Ssiz_t capacity, Ssiz_t nchar)
       Error("*TString::Init", "Negative length!");
       nchar = 0;
    }
-   if (nchar > capacity) {
-      nchar = capacity;
-   }
    if (capacity > MaxSize()) {
       Error("TString::Init", "capacity too large (%d, max = %d)", capacity, MaxSize());
       capacity = MaxSize();
+      if (nchar > capacity) {
```

Comment:
Just realized that the curly brackets are unnecessary. Should I remove them or just keep it like this?

Reply:
Keep it like it is please (less noise in the diffs).

Reply:
Okay

Reply:
argh sorry, I was mistaken, there were no brackets originally! So indeed let's keep it without -- sorry for the back and forth!

Reply:
No problem. I'll change it